setup.py

```diff
@@ -17,7 +17,7 @@
 
 
 package_name = "dbt-teradata-utils"
-package_version = "0.0.1a"
+package_version = "0.0.2a"
```

Comment:
which is the correct version string?

Reply:
it should be "0.0.1a". This string will be replaced automatically when the release workflow runs.

https://github.com/Teradata/dbt-teradata/blob/main/.github/workflows/cd-release.yml#L25

Reply:
I have changed it to "0.0.1a"